src/image/p5.Image.js

```diff
@@ -890,7 +890,7 @@ p5.Image.prototype.isModified = function() {
  */
 p5.Image.prototype.save = function(filename, extension) {
   if (this.gifProperties) {
-    p5.prototype.saveGif(this, filename);
+    p5.prototype.encodeAndDownloadGif(this, filename);
```

Comment:
🚧   this might be a serious breaking point, we have to take care of it    🚧

Reply:
I made sure to substitute the old `saveGif` function with the encodeAndDownloadGif, which is the original function that was already present in the repo. Even the tests were considered.

It indeed is risky though! I'll keep close attention to that!

Reply:
can you please make a gist of what this function is doing  and  how your replacement does not affects  this functionality in terms of programming.

also did you run every examples using this function, perhaps this would be present in examples folder!

Reply:
`encodeAndDownloadGif` is exactly the same function that was before, just that it was called `saveGif`. They call it as _private_, in the sense that the normal users won't know about the function. That's why it does not appear in the documentation